### DISCUSSION TITLE
Cardano bug

### DISCUSSION THREAD

**Discussion by DiDi-Chuxing** [IntersectMBO/cardano-node#5025] [General]
I found a very big bug when transferring ADA to another wallet address which causes the transfer to stop for 24 hours and ADA transactions to stop.
If this bug continues for 24 hours, Cardano and all the currencies of this network will stop working.
ADA will drop to $0.005 if I release this bug on public networks.
Do you want to know that bug?
👇
DiDi_Chuxing@outlook.com

**Comment by erikd** [comment]
I will bite. What is the bug?

**Comment by DiDi-Chuxing** [comment]
If you are one of Cardano's administrators, send me a message to my email so that I can tell you the bug and the solution to fix it.

**Reply by katomm** [reply]
What's wrong with using GitHub?

**Reply by erikd** [reply]
Email sent.

**Reply by erikd** [reply]
@DiDi-Chuxing I have sent you email, but you have not responded.

**Reply by DiDi-Chuxing** [reply]
> @DiDi-Chuxing I have sent you email, but you have not responded.

I received your email, but I said earlier that I will only tell Cardano administrators about the bug and the solution to fix it.

**Reply by erikd** [reply]
Well 3 days ago you said:

> If this bug continues for 24 hours, Cardano and all the currencies of this network will stop working.

Its been more than 48 hours and nothing has happened.

**Reply by DiDi-Chuxing** [reply]
> Well 3 days ago you said:
> 
> > If this bug continues for 24 hours, Cardano and all the currencies of this network will stop working.
> 
> Its been more than 48 hours and nothing has happened.

I said that if this happens, I have not used this bug.

**Reply by erikd** [reply]
@DiDi-Chuxing You should not be confused. You made what sounded like a threat with a deadline and you failed to follow through.

**Reply by erikd** [reply]
@DiDi-Chuxing I may not be an "Administrator", but I am pretty close to just deleting this whole discussion so that nobody on the Cardano team wastes any more time on this.

**Reply by DiDi-Chuxing** [reply]
> @DiDi-Chuxing You should not be confused. You made what sounded like a threat with a deadline and you failed to follow through.

I explained this bug to the Cardano administrators on Twitter and the website, but they may not have seen my message

**Reply by erikd** [reply]
Links?

**Reply by DiDi-Chuxing** [reply]
> Links?

https://twitter.com/ronja_jurvala/status/1607308662560948226?s=46&t=bGGNNaEGQ3PAZ2tsTkxweg

**Reply by erikd** [reply]
Just as little information as this "bug report". In other words, completely useless.

**Reply by erikd** [reply]
On February 4th this year, you claimed on Twatter that you sold information about this bug to a "big American investment company":
https://twitter.com/Ronja_Jurvala/status/1621722686173970433

If this was such a dangerous bug surely this "big American investment company" would have profited by now.

**Reply by DiDi-Chuxing** [reply]
I said that if no one wants the bug, I will sell it, why don't you pay attention to the end of the sentences?

**Reply by erikd** [reply]
> I said that if no one wants the bug, I will sell it, why don't you pay attention to the end of the sentences?

No, you said in the tweet i referenced: "We sold the bug to a big investment company in America, we hope they will use it properly."

**Reply by DiDi-Chuxing** [reply]
Maybe the translator translated the wrong words, we said we will sell it, not that we have sold it.
Anyway
My attempt to report a bug is useless.  I will sell the bug and the bug fix solution to the same investment company.

**Reply by DiDi-Chuxing** [reply]
> > I said that if no one wants the bug, I will sell it, why don't you pay attention to the end of the sentences?
> 
> No, you said in the tweet i referenced: "We sold the bug to a big investment company in America, we hope they will use it properly."

We meant selling it to an American investment company in the future, which may have been a typo, but I deleted that tweet anyway.

**Reply by erikd** [reply]
> I deleted that tweet anyway.

Nevermind, I have a screenshot.


![big-american-investment-company-small](https://user-images.githubusercontent.com/307277/229029028-939a2060-ef6d-476f-ad56-3fc3b88363aa.png)

**Comment by newhoggy** [comment]
I'm an admin.  I will email you.

**Comment by erikd** [comment]
Closing this as  a stupid waste of time.
